docs/sources/dashboards/use-dashboards.md

```diff
@@ -77,39 +73,76 @@ By hovering over a panel with the mouse you can use some shortcuts that will tar
 - `pr`: Remove Panel
 - `pl`: Toggle panel legend
 
-## Dashboard Search
-
-Dashboards can be searched by the dashboard name, filtered by one (or many) tags or filtered by starred status. The dashboard search is accessed through the dashboard picker, available in the dashboard top nav area. The dashboard search can also be opened by using the shortcut `F`.
-
-{{< figure src="/static/img/docs/v50/dashboard_search_annotated.png" width="700px" >}}
+## Dashboard search
 
-1. `Search Bar`: The search bar allows you to enter any string and search both database and file based dashboards in real-time.
-1. `Starred`: Here you find all your starred dashboards.
-1. `Recent`: Here you find the latest created dashboards.
-1. `Folders`: The tags filter allows you to filter the list by dashboard tags.
-1. `Root`: The root contains all dashboards that are not placed in a folder.
-1. `Tags`: The tags filter allows you to filter the list by dashboard tags.
+You can search for dashboards by dashboard name and by panel title. When you search for dashboards, the system returns all dashboards available within the Grafana instance, even if you do not have permission to view the contents of the dashboard.
 
-When using only a keyboard, you can use your keyboard arrow keys to navigate the results, hit enter to open the selected dashboard.
+### Search dashboards using dashboard name
 
-### Find by dashboard name
-
-Begin typing any part of the desired dashboard names in the search bar. Search will return results for any partial string match in real-time, as you type.
+Begin typing any part of the dashboard name in the search bar. The search returns results for any partial string match in real-time, as you type.
 
 Dashboard search is:
 
 - Real-time
 - _Not_ case sensitive
 - Functional across stored _and_ file based dashboards.
 
-### Filter by Tag(s)
+> **Note: You can use your keyboard arrow keys to navigate the results and press `Enter` to open the selected dashboard.
+
+The following image shows the search results when you search using dashboard name.
+
+{{< figure src="/static/img/docs/v91/dashboard-features/search-by-dashboard-name.png" width="700px" >}}
+
+### Search dashboards using panel title
+
+You can also search for a dashboard by the title of a panel that appears in a dashboard.
```

Comment:
🤔 I think we should add that from 9.1 it is available by default on Grafana Cloud, and on OSS it can be enabled using the panelTitleSearch feature toggle.